src/main/java/se/liu/ida/hefquin/engine/queryplan/logical/impl/LogicalOpBGPOptAdd.java

```diff
@@ -91,4 +91,5 @@ public String toString(){
 				+ " )";
 	}
 
+
```

Comment:
Can you please try to avoid adding such spurious empty lines if they have not been there before.

```suggestion
```